www/front_src/src/Resources/Listing/columns/State/DetailsTable/Acknowledgement.tsx

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable react/prop-types */
```

Comment:
why this ?

Reply:
I guess a regression in the latest eslint-plugin-react that doesn't handle properly the Props signature. I'll investigate on that later on

Reply:
Ok